examples/linear_model/plot_lasso_lars_ic.py

```diff
@@ -0,0 +1,114 @@
+"""
+==============================================
+Lasso model selection via information criteria
+==============================================
```

Comment:
Could we merge both examples or is it better to keep them separate?
I'm asking as I notice a grow of examples which makes the gallery confusing.

Reply:
I would prefer to have them separated. I am afraid that there are too many messages to convey (cross-validation vs. AIC/BIC + explanation of AIC/BIC) if we merge the example. In addition, we only have a single example for `LassoLarsIC` up to now. It is a bit less problematic than if we would have already had several examples there.

Reply:
I would say `LassoLarsIC` qualifies for a single example :smirk: I'm really more concerned about too many examples (some of them old and not of the quality of more recent ones like in this PR).

Reply:
@ogrisel @agramfort What do you think about this?

Reply:
I can live with one more example that makes a stronger focus on the BIC/AIC models